README.md

```diff
@@ -315,16 +266,260 @@ For example:
   - `fields, fields->>'c2', sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields, sqrt((fields->>'c1')::int)`: function `sqrt()` is not pushed down.
   - `fields->>'c2', sqrt((fields->>'c1')::int)`: there is no fields jsonb column, so function `sqrt()` is pushed down.
+  
+Also see [Limitations](#limitations).  
+
+Supported platforms
+-------------------
 
-## Contributing
+`influxdb_fdw` was developed on Linux, and should run on any
+reasonably POSIX-compliant system.
+
+`influxdb_fdw` is designed to be compatible with PostgreSQL 10 ~ 14.
+
+Go version should be 1.10.4 or later.
+
+Installation
+------------
+### Prerequisites
+
+Install InfluxDB Go client library
+<pre>
+go get github.com/influxdata/influxdb1-client/v2
+</pre>
+
+### Source installation
+
+Add a directory of pg_config to PATH and build and install influxdb_fdw.
+<pre>
+make USE_PGXS=1 with_llvm=no
+make install USE_PGXS=1 with_llvm=no
+</pre>
+with_llvm=no is necessary to disable llvm bit code generation when PostgreSQL is configured with --with-llvm because influxdb_fdw use go code and cannot be compiled to llvm bit code.
+
+If you want to build influxdb_fdw in a source tree of PostgreSQL instead, use
+<pre>
+make with_llvm=no
+make install  with_llvm=no
+</pre>
+
+Usage
+-----
+
+## CREATE SERVER options
```

Comment:
Could you please update the new options as well?
for using cxx-client, 'version' for influxdb version and 'retention_policy' while connect to InfluxDB ver 2.x

----

#### Go Client connect to InfluxDB ver 1.x
<pre>
CREATE SERVER influxdb_server FOREIGN DATA WRAPPER influxdb_fdw OPTIONS
(dbname 'mydb', host 'http://localhost', port '8086') ;
</pre>
#### Influxdb_cxx Client connect to InfluxDB ver 1.x
<pre>
CREATE SERVER influxdb_server FOREIGN DATA WRAPPER influxdb_fdw OPTIONS
(dbname 'mydb', host 'http://localhost', port '8086', version '1') ;
</pre>
#### Influxdb_cxx Client connect to InfluxDB ver 2.x
<pre>
CREATE SERVER influxdb_server FOREIGN DATA WRAPPER influxdb_fdw OPTIONS
(dbname 'mydb', host 'http://localhost', port '8086', version '2', retention_policy '') ;
</pre>

Reply:
Some descriptions added in USAGE section. Examples added in EXAMPLE section near `CREATE SERVER` SQL.

Reply:
Thank you for adding both side of option description and examples